modular_skyrat/master_files/code/modules/clothing/towels.dm

```diff
@@ -38,6 +41,7 @@
 	item_flags = NOBLUDGEON
 	resistance_flags = FLAMMABLE
 	flags_inv = SHOWSPRITEEARS // Only relevant when in head shape, but useful to keep around regardless.
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION
```

Comment:
Just to make 10000% sure, this doesn't try to do anything fucky when it's in head shape?